### DISCUSSION TITLE
Running build - Did you mean to import glamor/lib/hash.js?

### DISCUSSION THREAD

**Discussion by ross-hulford** [nuxt/nuxt#16961] [Questions]
**"nuxt": "3.0.0-rc.6",**

Running the **yarn build** then **node .output/server/index.mjs**

I get the error 

**Did you mean to import glamor/lib/hash.js?**

Another thread led me to try this

```
export default defineNuxtConfig({
  experimental: { viteNode: true },
  typescript: {
    strict: true,
  },
  build: {
    transpile: ['lodash', ' glamor'],
  },
```

```
node -v
v16.15.1
```

**Comment by TheAlexLichter** [comment]
Could you kindly add a reproduction via StackBlitz / CodeSandbox (see https://nuxt.new/ for templates)
